integration-tests/src/test/java/org/apache/fineract/integrationtests/ClientLoanIntegrationTest.java

```diff
@@ -5490,7 +5490,7 @@ public void chargeAdjustmentAccountingValidation() {
                     .withDaysInMonth("30").withDaysInYear("365").withMoratorium("0", "0")
                     .withFeeToIncomeAccountMapping(feeCharge.getResourceId(), uniqueIncomeAccountForFee.getResourceId())
                     .withPenaltyToIncomeAccountMapping(penaltyCharge.getResourceId(), uniqueIncomeAccountForPenalty.getResourceId())
-                    .withFeeAndPenaltyAssetAccount(assetFeeAndPenaltyAccount).build(null);
+                    .withFeeAndPenaltyAssetAccount(assetFeeAndPenaltyAccount).withChargeOffReasonsToExpenseMappings(1L, 12L).build(null);
```

Comment:
This seems a little random. Would you mind to remove?

Reply:
Removed